vault/values.yaml

```diff
@@ -0,0 +1,62 @@
+vault:
+  devMode: false
+  port: 8200
+  clusterPort: 8201
+  repository: vault
+  tag: 1.2.1
+  pullPolicy: IfNotPresent
+  volumeSize: 5Gi
+  secretsPath: secrets/smart_city
+  liveness:
+    initialDelaySeconds: 30
+    periodSeconds: 10
+  readiness:
+    initialDelaySeconds: 10
+    periodSeconds: 10
+  config:
+    listener:
+      tcp:
+        address: '[::]:8200'
+        cluster_address: '[::]:8201'
+        tls_disable: true
+        tls_prefer_server_cipher_suites: true
+        tls_cipher_suites: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA
```

Comment:
Is this leaking anything to include?

Reply:
I do not believe so.  We would need to put at least some ciphers here per my understanding and none of these seemed like "bad" ciphers